README.rst

```diff
@@ -29,7 +29,7 @@ Features
 
 - Built-in support for a wide variety of languages, including Ada, C, C#, C++,
   Haskell, Java, JavaScript, Lua, OCaml, Pascal, Perl, PHP, Python, Ruby,
-  Scheme…
+  Scheme, Scala…
```

Comment:
The list is sorted, please put Scheme last :-)

Reply:
Dame it